mopidy/local/json.py

```diff
@@ -128,8 +90,18 @@ def browse(self, uri):
     def load(self):
         logger.debug('Loading library: %s', self._json_file)
         with timer.time_logger('Loading tracks'):
-            library = load_library(self._json_file)
-            self._tracks = dict((t.uri, t) for t in library.get('tracks', []))
+            if not os.path.isfile(self._json_file):
+                logger.info(
+                    'No local library metadata cache found at %s. Please run '
+                    '`mopidy local scan` to index your local music library. '
+                    'If you do not have a local music collection, you can '
+                    'disable the local backend to hide this message.',
+                    self._json_file)
+                self._tracks = {}
+            else:
+                library = internal.storage.load(self._json_file)
```

Comment:
Does `internal.storage` work? We've only imported `internal` and I don't want that to imply an import of `internal.storage`. I see the namespace collision with `mopidy.local.storage` that motivated this. I suggest to instead use an aliased import:

``` python
from mopidy.internal import storage as internal_storage
```